coverage_excludes.txt

```diff
@@ -0,0 +1,12 @@
+# This is a simple file containing regx patterns that should be
```

Comment:
It's good that the file explains it is a simple file.

Reply:
Will write a less stupid comment :slightly_smiling_face: